dataclients/kubernetes/ingress.go

```diff
@@ -318,17 +318,23 @@ func (ing *ingress) addEndpointsRule(ic ingressContext, host string, prule *defi
 		redirect.setHostDisabled(host)
 	}
 
+	if ing.kubernetesEnableEastWest {
+		ewRoute := createEastWestRouteIng(ing.eastWestDomainRegexpPostfix, ic.ingress.Metadata.Name, ic.ingress.Metadata.Namespace, endpointsRoute)
+		ic.addHostRoute(ewRoute.HostRegexps[0], ewRoute)
```

Comment:
I think we should use the host and not the host regexp as the key here. See [line 350](https://github.com/zalando/skipper/pull/1630/files#diff-ed72f00dd7cffd2b4c074148c2e982ec17d983aa243e312b4552b3485325eb89R350), and also the test with [catchall](https://github.com/zalando/skipper/pull/1630/files#diff-ac2da4b4414349fd8688b4a28a94d7e5f8dc9494ab2328866f570c62066b68adR23)

Reply:
true